docs/next/styles/globals.css

```diff
@@ -99,6 +99,16 @@ dt > a.reference.internal {
   @apply absolute top-3 right-3 opacity-50 text-sm;
 }
 
+.experimental-tag,
+.flag.experimental {
+  @apply inline-flex items-center px-3 py-0.5 rounded-full align-middle text-xs uppercase font-medium bg-sea-foam text-gable-green
```

Comment:
This combines Tailwind classes together to be applied to the `.experimental-tag` and `.flag.experimental` selectors.